README.md

```diff
@@ -178,6 +165,43 @@ hcl/dx/image-processor:v1.13.0_20211213-1446               "/home/dx_user/start
 
 ## Tips and tricks
 
+### Performance Tuning
+
+The performance for local docker volumes on some operating systems like Mac OS and Windows is quite slow.
+To improve especially the startup time of DX Core, you may choose to remove the persistent volume configuration.
+To do so, remove the following lines from the docker-compose file:
+
+```bash
+    volumes:
+      - ./volumes/core/wp_profile:/opt/HCL/wp_profile
+```
+
+> **_NOTE:_** When removing the volumes at all, any change you apply in DX Core will not be persisted. All your changes will be lost as soon as the container is stopped. If you need any specific folders or files been persistent, then please try to reduce it as much as possible. For example by just saving the server config of the WebSphere_Portal JVM to your local volume.  
+
+### Debugging WebSphere_Portal JVM on dx-core container
+
+It is possible to enable debugging mode on the WebSphere_Portal JVM to monitor/debug the whole server process.  
+For that, please make sure to enable port 7777 for the core service and in your firewall. Below you will find an example to expose that port 7777 in the dx.yaml file:  
+
+```bash  
+    services:  
+      core:  
+        container_name: dx-core  
+        image: ${DX_DOCKER_IMAGE_CORE:?'Missing docker image environment parameter'}  
+        ports:  
+          - "7777:7777"  
+```
+
+In addition to that it is needed to enable the WebSphere Application Server **debugging services** in the IBM Integrated Solutions Console (admin console). For details, please check: [Debugging Service details](https://www.ibm.com/docs/en/was/9.0.5?topic=applications-debugging-service-details)
+
+> **_NOTE:_** Make sure that the changes are persistent for the current dx-core container. A WebSphere_Portal server restart is needed, as soon as the debugging services are enabled!  
+
+After debugging is enabled you can use any IDE like Microsoft Visual Studio Code, Eclipse or IBM Rational Application Developer to connect to that remote debugging port. For details, please check: [HDX-DEV-300 HCL Digital Experience for Developers (Advanced)](https://hclsoftwareu.hcltechsw.com/courses/course/hdx-dev-300-dx-developer-advanced)  
+
+### Adding Shared Libraries
+
+HCL highly recommand to add shared libraries in the folder `/opt/HCL/wp_profile/PortalServer/sharedLibrary/` of the dx-core container to avoid performance problems with shared libraries.  
```

Comment:
```suggestion
HCL highly recommends to add shared libraries in the folder `/opt/HCL/wp_profile/PortalServer/sharedLibrary/` of the dx-core container to avoid performance problems with shared libraries.  
```

Reply:
changed as suggested